inference-engine/include/cldnn/cldnn_config.hpp

```diff
@@ -79,10 +79,12 @@ DECLARE_CLDNN_CONFIG_KEY(NV12_TWO_INPUTS);
 DECLARE_CLDNN_CONFIG_KEY(MAX_NUM_THREADS);
 
 /**
-* @brief This key enable loop unrolling for recurrent layers such as LSTM, GRU and TensorIterator.
-* By default the graph optimization unrolls TensorIterator layers.
-* If the ENABLE_LOOP_UNROLLING is turn off, TensorIterator body is separated to sub-program instead of loop unrolling,
-* compile it once, and then execute it with proper inputs and outputs for each iteration.
+* @brief Turning on this key enables to unroll recurrent layers such as TensorIterator or Loop with fixed iteration count.
+* This key is turned on by default. Note that unrolling loops will cause increase of clDNN build time,
+* because the kernels consiting the body network are created and built for all iterations.
```

Comment:
* `consiting` -> `consisting`. By the way, I don't think `consist` is the proper verb here because it is intransitive verb. What about "because the kernels that forms the body network"?

Reply:
Thanks, fixed.